src/message-io.js

```diff
@@ -62,7 +64,20 @@ module.exports = class MessageIO extends EventEmitter {
   }
 
   startTls(credentialsDetails: Object, hostname: string, trustServerCertificate: boolean) {
-    const credentials = tls.createSecureContext ? tls.createSecureContext(credentialsDetails) : crypto.createCredentials(credentialsDetails);
```

Comment:
`tls.createSecureContext` is available in all Node.js versions that we support, so I removed this conditional here.